src/solver/optimisation/adequacy_patch.h

```diff
@@ -28,29 +28,74 @@
 #ifndef __SOLVER_ADEQUACY_FUNCTIONS_H__
 #define __SOLVER_ADEQUACY_FUNCTIONS_H__
 
+typedef std::pair<AdequacyPatchMode, AdequacyPatchMode> adqPair;
+
+namespace Antares
+{
+namespace Data
+{
+namespace AdequacyPatch
+{
+
```

Comment:
I do not think this is what his comment wants. For me, he wants to remove "using namespace Antares::Data::AdequacyPatch", then write "AdequacyPatch::adqmPhysicalAreaInsideAdqPatch" instead of "adqmPhysicalAreaInsideAdqPatch".
Again, I do not think what he wants is more clear than the previous version.